package.json

```diff
@@ -25,9 +25,6 @@
     "geojsonhint": "0.3.4",
     "gtfs2geojson": "^1.0.0",
     "jsonlint": "1.6.0",
-    "leaflet-draw": "0.2.3",
-    "leaflet-geodesy": "0.1.1",
-    "leaflet-hash": "0.2.1",
     "mapbox.js": "2.2.2",
```

Comment:
If even the mobile site is loading gl.js, should we remove this?

Reply:
Yes, we should... pushed a commit to remove mapbox.js just now.